lucille-examples/lucille-distributed-example/conf/simple-pipelines.conf

```diff
@@ -0,0 +1,29 @@
+# this illustrates how a list of pipeline definitions be placed in a separate file and referenced from elsewhere
```

Comment:
let's move the pipeline definitions into the main config file
they were only placed in a separate config in the file-to-file-example to illustrate how that could be done, but there's no need to illustrate that again here